utils/fetcher.ts

```diff
@@ -20,7 +20,9 @@ export const getErrorMessage = async (res: Response) => {
   let messageError = res.statusText
   try {
     messageError = `${res.statusText}: ${(await res.json()).message}`
-  } catch (e) {}
+  } catch (e) {
+    // unable to identify error message, possibly a network failure
```

Comment:
Should we not log or return something here instead of just a comment

Reply:
Nah, it will then use the default message `res.statusText` which is fine for handling that error.